main/glossary/README.md

```diff
@@ -67,151 +104,379 @@ synchronously compared for structural equivalence.
 
 A comparable is a JavaScript object containing no promises, and can
 thus be locally compared for equality with another object. If either object
-contains promises, equality is indeterminable. If both are fulfilled down
+contains Promises, equality is indeterminable. If both are fulfilled down
 to Presences and local state, then either they're the same all the way
 down, or they represent different objects.
 
 ## Contract and Contract Instance
-In Agoric documentation, *contract* usually refers to a contract's source code that defines how the contract works, which is *installed* on Zoe. A contract is *instantiated* to create *contract instances*, which are run on Zoe and are the active execution of the contract code.  
+In Agoric documentation, *contract* usually refers to a contract's source code that 
+defines how the contract works. A contract's source code is *installed* on Zoe. A 
+contract is *instantiated* to create *contract instances*, which are the active 
+execution of a contract's code running on Zoe.  
+
+For example, a realtor has a standard house selling agreement. The contract is the 
+code defining how that agreement works. When the realtor has a new house to sell, 
+they instantiate a new instance of their standard contract for that specific property. 
+If they have ten houses for sale, they have ten different contract instances.
+
+## CreatorInvitation
+
+An [invitation](#invitation) optionally returned by `startInstance()` that the contract instance
+creator can use. It is usually used in contracts where the creator immediately 
+sells something (auctions, swaps, etc.). 
```

Comment:
Honestly pretty sure I was told the deleted at some point. Is it definitely not the case, or should I maybe include both reasons?

Reply:
That is a circumstance in which the creatorInvitation is usually used, but it's not the case that most of the circumstances in which the creatorInvitation is available are because the creator will sell something immediately.  

Just a change in emphasis.  Most As are B, but that doesn't mean that most Bs are A.

"If the creator will usually want to immediately sell something, the contract should probably provide a creatorFacet."